consensus/abci.go

```diff
@@ -94,127 +89,110 @@ func (abci *TendermintABCI) Info(req tmtAbciTypes.RequestInfo) tmtAbciTypes.Resp
 	}
-	abci.logger.Info("DeliverTx: Received valid transaction", "tx", tx.Hash().String()) // nolint: errcheck
 
 	res := abci.db.DeliverTx(tx)
 	if res.IsErr() {
-		// nolint: errcheck
-		abci.logger.Error("DeliverTx: Error delivering tx to database db", "tx", tx.Hash().String(),
-			"err", err)
+		abci.logger.Error("DeliverTx: Error delivering tx to database db", "tx", tx.Hash().String(), "err", err)
 		return res
 	}
 
 	abci.CollectTx(tx)
+
+	abci.logger.Info("TX delivered.", "tx", tx.Hash().String())
+
 	return tmtAbciTypes.ResponseDeliverTx{Code: tmtAbciTypes.CodeTypeOK}
 }
 
 // BeginBlock starts a new Ethereum block
 func (abci *TendermintABCI) BeginBlock(req tmtAbciTypes.RequestBeginBlock) tmtAbciTypes.ResponseBeginBlock {
-	abci.logger.Info("TendermintABCI::BeginBlock()")
-	abci.logger.Debug("BeginBlock") // nolint: errcheck
-
-	// update the eth header with the tmtCfg header
 	abci.db.UpdateHeaderWithTimeInfo(&req.Header)
 	return tmtAbciTypes.ResponseBeginBlock{}
 }
 
 // EndBlock accumulates rewards for the validators and updates them
 func (abci *TendermintABCI) EndBlock(req tmtAbciTypes.RequestEndBlock) tmtAbciTypes.ResponseEndBlock {
-	abci.logger.Info("TendermintABCI::EndBlock()")
-	//abci.db.AccumulateRewards(abci.strategy)
 	return tmtAbciTypes.ResponseEndBlock{}
 }
 
 // Commits the block and returns a hash of the current state
 func (abci *TendermintABCI) Commit() tmtAbciTypes.ResponseCommit {
 	blockHash, err := abci.db.Commit(abci.Receiver())
 	if err != nil {
-		// nolint: errcheck
 		abci.logger.Error("Error getting latest database state", "err", err)
+		return tmtAbciTypes.ResponseCommit{}
 	}
 
 	ethState, err := abci.getCurrentDBState()
 	if err != nil {
-		abci.logger.Error("Error getting latest state", "err", err) // nolint: errcheck
+		abci.logger.Error("Error getting latest state", "err", err)
+		return tmtAbciTypes.ResponseCommit{}
 	}
 
-	abci.logger.Info("TendermintABCI::Commit()", "blockHash", blockHash.Hex())
+	abci.logger.Info("Committing state", "blockHash", blockHash.Hex())
+
 	rootHash := abci.getCurrentBlock().Root()
 	abci.checkTxState = ethState.Copy()
 	if err != nil {
-		abci.logger.Error("Error committing latest state", "err", err) // nolint: errcheck
+		abci.logger.Error("Error committing latest state", "err", err)
+		return tmtAbciTypes.ResponseCommit{}
 	}
-	// The app should respond to the Commit request with a byte array, which is the deterministic state root of the
-	// application. It is included in the header of the next block. It can be used to provide easily verified
-	// Merkle-proofs of the state of the application.
+
 	return tmtAbciTypes.ResponseCommit{Data: rootHash.Bytes()}
 }
 
 // Query queries the state of the TendermintABCI
 func (abci *TendermintABCI) Query(query tmtAbciTypes.RequestQuery) tmtAbciTypes.ResponseQuery {
-	abci.logger.Info("TendermintABCI::Query()", "data", query)
-	abci.logger.Debug("Query") // nolint: errcheck
+	abci.logger.Info("Querying state", "data", query)
```

Comment:
You removed similar log outputs, what did you decide to keep this one? I would be inclined to replace it, as mentioned above, with `Debug()`

Reply:
Didn't remove, added more logging even but on a different line :D